layers/core_validation.cpp

```diff
@@ -1102,20 +1346,6 @@ bool CoreChecks::ValidatePipelineDrawtimeState(const LAST_BOUND_STATE &state, co
         }
     }
 
-    // Verify if using dynamic state setting commands that it doesn't set up in pipeline
-    CBDynamicFlags invalid_status(~CBDynamicFlags(0));
-    invalid_status &= ~pCB->dynamic_status;
-    invalid_status &= ~pCB->static_status;
-
-    if (invalid_status.any()) {
-        std::string dynamic_states = DynamicStateString(invalid_status);
-        LogObjectList objlist(pCB->commandBuffer());
-        objlist.add(pPipeline->pipeline());
-        skip |= LogError(objlist, vuid.dynamic_state_setting_commands,
-                         "%s: %s doesn't set up %s, but it calls the related dynamic state setting commands", caller,
-                         report_data->FormatHandle(state.pipeline_state->pipeline()).c_str(), dynamic_states.c_str());
-    }
-
     // Verify vertex binding
     if (pPipeline->vertex_input_state && pPipeline->vertex_input_state->binding_descriptions.size() > 0) {
```

Comment:
Nitpick: Redundant size check given the for loop it's guarding.

```suggestion
    if (pPipeline->vertex_input_state) {
```